.travis.yml

```diff
@@ -39,10 +52,11 @@ install:
   - npm --version
   - npm install --progress false --depth 0
   - npm prune
+  - cd node_modules/mv
+  - npm install rimraf@~2.4.0
```

Comment:
This part is gross and likely to fall out of sync if/when the other packages that require it change dependencies. We still may as well land it in the name of progress, but can you think of a test case we could make to submit a bug to npm and/or the libraries that pull in rimraf to get rid of the need for this?  Then once we have a clean `npm ls` we should enforce that as part of the test suite to make sure we don't check in / update any packages that will re-introduce this issue.